Makefile

```diff
@@ -147,23 +155,23 @@ clean/deps:
 	rm -Rf bin/*
 
 .PHONY: deps
-deps: bin/go-enum bin/protoc-gen-go bin/protoc-gen-go-grpc bin/golangci-lint bin/goverter bin/openapi-generator-cli
+deps: bin/go bin/protoc bin/go-enum bin/protoc-gen-go bin/protoc-gen-go-grpc bin/golangci-lint bin/goverter bin/openapi-generator-cli
 
 .PHONY: vendor
 vendor:
-	go mod vendor
+	${GO} mod vendor
 
 .PHONY: build
 build: gen vet lint
-	go build
+	${GO} build -buildvcs=false
 
 .PHONY: build/odh
 build/odh: vet
-	go build
+	${GO} build -buildvcs=false
```

Comment:
Was `-buildvcs=false` needed only for the build-in-docker? 🤔 Because on my end the `make build` works without issue as-is, without needing ` -buildvcs=false`

Reply:
it fails in CI without this flag ...